src/Cupboard.Providers.Windows/Chocolatey/ChocolateyPackageProvider.cs

```diff
@@ -28,12 +28,34 @@ public override ChocolateyPackage Create(string name)
 
         protected override bool IsPackageInstalled(ChocolateyPackage resource, string output)
         {
-            return output.Contains(resource.Package, StringComparison.OrdinalIgnoreCase);
+            var containsPackage = output.Contains(resource.Package, StringComparison.OrdinalIgnoreCase);
+            if (string.IsNullOrEmpty(resource.PackageVersion))
+            {
+                return containsPackage;
+            }
+
+            if (!containsPackage)
+            {
+                return false;
+            }
+
+            var indexOfPipe = output.IndexOf('|');
+            var indexOfNewLine = output.IndexOfAny(new[] { '\r', '\n' }, indexOfPipe);
+            if (indexOfPipe == -1 || indexOfNewLine == -1)
+            {
+                return false;
+            }
+
+            var versionString = output.Substring(indexOfPipe + 1, indexOfNewLine);
+            return Version.TryParse(versionString, out var currentPackageVersion)
+                && Version.TryParse(resource.PackageVersion, out var packageVersion)
+                && currentPackageVersion >= packageVersion;
         }
 
         protected override async Task<ProcessRunnerResult> GetPackageState(ChocolateyPackage resource)
         {
-            return await _runner.Run("choco", "list -lo", supressOutput: true).ConfigureAwait(false);
+            var arguments = $"list --limit-output --local-only {resource.Package}";
```

Comment:
This produces the following output:

```
$ choco list -l --limit-output sops
sops|3.7.1

```

Reply:
Looks good to me. Go with the new output as you suggested!